charon/src/transform/index_to_function_calls.rs

```diff
@@ -132,12 +128,9 @@ impl<'a> IndexVisitor<'a> {
             );
             self.statements.push(Statement::new(self.span, kind));
 
-            // `storage_live(index_var)`
```

Comment:
Let's keep these comments, they are accurate even if the `StorageLive` is added inside `fresh_var`